recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,272 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - h5py >=2.2,<2.8
+    - numpy >=1.8,<1.12
+    - scipy >=0.13,<0.18
```

Comment:
@mhearne-usgs I am pretty sure that the bad combinations of packages are the limitations on `numpy` and `scipy` here. If we can use a newer version of them everything should work on Python 3.6 too.

If not we need to release older versions of those to Python 3.6 or skip Python 3.6 builds.